apps/amiapp_flutter/android/app/build.gradle

```diff
@@ -7,10 +7,13 @@ plugins {
 
 android {
     namespace 'io.customer.amiapp_flutter'
-    compileSdkVersion 33
+    compileSdkVersion 34
     ndkVersion flutter.ndkVersion
 
     compileOptions {
+        // Flag to enable support for the new language APIs
+        coreLibraryDesugaringEnabled true
```

Comment:
interesting, what new language API are we introducing?

Reply:
This is required by [flutter_local_notifications gradle setup](https://pub.dev/packages/flutter_local_notifications#gradle-setup).